### ISSUE TITLE
Finetuning C3D on UCF101

### ISSUE THREAD

**Issue by skylian** [facebook/C3D#89]
I'm following the configuration provided in the example, but with a few exceptions:
1. Batch size. 12GB GPU memory is not large enough for a batch size of 30. I have to reduce the batch size to 10. 
2. I tripled the step size and max number of iterations, and lower the learning rate to one third to compensate the smaller batch size.
3. Looks like the ffmpeg on my machine always extracts more frames from a video. I think that is the problem of the codecs. But I'm finetuning so I guess that would not matter.

The clip accuracy peaks at iteration 24000, which is 75%, then it starts going down. Most of the time, the training loss looks normal, but sometimes I observe very high loss for some batches, \eg over 10 while the others are below one. The test loss is always above 1.4.

Does 75% clip accuracy looks normal, or it can be higher?

**Comment by Benuraj** [comment]
This is the exact same Issue I'm facing too.

**Comment by dutran** [comment]
@skylian the loss should be much smaller and the accuracy should be above 80% (If i remember correctly).

**Comment by skylian** [comment]
I suspect the problem is in video codecs on my machine. I found a lot of consecutive frames that look exactly the same, which results in more frames than you. I managed to get another codes which produces same amount of frames as you. I'm waiting to see the final results.

**Comment by dutran** [comment]
@skylian have you solved your issue?

**Comment by skylian** [comment]
Yes. Sorry I should have replied earlier.

On 31 March 2016 at 22:41, Du Tran <notifications@github.com> wrote:

> @skylian <https://github.com/skylian> have you solved your issue?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/C3D/issues/89#issuecomment-204255216>
>



-- 
Lian, Xiaochen
Department of Statistics
University of California, Los Angeles

**Comment by dutran** [comment]
Glad to hear! Good luck with your experiments.
